### ISSUE TITLE
Licensing issues with iTK 5.1.0

### ISSUE THREAD

**Issue by badshah400** [InsightSoftwareConsortium/ITK#1913]
Some files in VXL that ITK 5.1.0 uses have non-free licenses, making them incompatible with ITK's Apache-2.0 Licensing:

* Modules/ThirdParty/VNL/src/vxl/v3p/netlib/triangle.c:
```
/*  This program may be freely redistributed under the condition that the    */
/*    copyright notices (including this entire header and the copyright      */
/*    notice printed when the `-h' switch is selected) are not removed, and  */
/*    no compensation is received.  Private, research, and institutional     */
/*    use is free.  You may distribute modified versions of this code UNDER  */
/*    THE CONDITION THAT THIS CODE AND ANY MODIFICATIONS MADE TO IT IN THE   */
/*    SAME FILE REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR, BOTH SOURCE   */
/*    AND OBJECT CODE ARE MADE FREELY AVAILABLE WITHOUT CHARGE, AND CLEAR    */
/*    NOTICE IS GIVEN OF THE MODIFICATIONS.  Distribution of this code as    */
/*    part of a commercial system is permissible ONLY BY DIRECT ARRANGEMENT  */
/*    WITH THE AUTHOR.  (If you are not directly supplying this code to a    */
/*    customer, and you are instead telling them how they can obtain it for  */
/*    free, then you are not required to make any arrangement with me.)      */
```

* Modules/ThirdParty/VNL/src/vxl/v3p/netlib/examples/showme.processed.c (same licensing terms as in 1 above)
* Modules/ThirdParty/VNL/src/vxl/v3p/netlib/triangle.README
```
These programs may be freely redistributed under the condition that the
copyright notices (including the copy of this notice in the code comments
and the copyright notice printed when the `-h' switch is selected) are
not removed, and no compensation is received.  Private, research, and
institutional use is free.  You may distribute modified versions of this
code UNDER THE CONDITION THAT THIS CODE AND ANY MODIFICATIONS MADE TO IT
IN THE SAME FILE REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR, BOTH
SOURCE AND OBJECT CODE ARE MADE FREELY AVAILABLE WITHOUT CHARGE, AND
CLEAR NOTICE IS GIVEN OF THE MODIFICATIONS.  Distribution of this code as
part of a commercial system is permissible ONLY BY DIRECT ARRANGEMENT
WITH THE AUTHOR.  (If you are not directly supplying this code to a
customer, and you are instead telling them how they can obtain it for
free, then you are not required to make any arrangement with me.)
```

These non-free licenses make it impossible to ship the updated ITK as part of openSUSE, see <https://build.opensuse.org/request/show/817960>. See also parallel vxl issue https://github.com/vxl/vxl/issues/752.

**Comment by badshah400** [comment]
The  [NOTICE](../blob/master/NOTICE) file says (line number 109):
```
7) The triangle.c/h and showme.c files have been removed.
                  Their copyright/license was incompatible with
                  ITK and their functionality was not needed.
                  These removed the "itknetlib" library.
```
but the Statistics module depends on ITKnetlib which requires the non-free `triangle.*` files in `Modules/ThirdParty/VNL/src/vxl/v3p/netlib/`.

**Comment by thewtex** [comment]
@badshah400 thanks for the note. I'll see if we can remove these from the repository.

CC @hjmjohnson

**Comment by thewtex** [comment]
Patch in #1920, to be included in v5.1.1.

**Comment by badshah400** [comment]
Super, thanks!
